README.md

```diff
@@ -1,57 +1,11 @@
-# Project Name
+# Federated Learning in Azure ML
 
-(short, 1-3 sentenced, description of the project)
+Although there is no specific commitment to a timeline yet, Azure ML is working on first-class support for cross-silo federated learning.
```

Comment:
Nit: suggest removing this and all other internal-only links.

Also, suggest adding:

- a bit more detail on what a "silo" means.
- a disclaimer that this repo is examples + recipes _only_; none of this has SLAs on support or correctness

Reply:
updated.